### PR TITLE
Doc number of search results is not available with secure search

### PR DESCRIPTION
**Pull request by sir-sigurd**

### CODE REVIEW

**Review by nl0 — APPROVED** [review]

File: docs/advanced-features/s3-prefix-permissions.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ key that the user is not allowed to access will display an _Access Denied_ messa
 * Prefix-aware search performs a head request on every object result; this may slow
 search performance in the Catalog
 
+* Number of search results is not available with prefix-aware search
```
Comment (akarve) [inline docs/advanced-features/s3-prefix-permissions.md:45]:
```suggestion
* The Catalog does not (accurately) display the number of search results
```

Comment (sir-sigurd) [reply docs/advanced-features/s3-prefix-permissions.md:45]: this limitation applies only to prefix-aware search aka secure search and you suggestion remove its mention

**Comment by sir-sigurd** [comment]
@nl0 
this is still correct, right?

**Comment by nl0** [comment]
> @nl0 this is still correct, right?

yep

**Pull request merged**